packages/aws-cdk-lib/aws-appsync/lib/graphqlapi.ts

```diff
@@ -312,6 +312,13 @@ export interface GraphqlApiProps {
    */
   readonly xrayEnabled?: boolean;
 
+  /**
+   * A value indicating whether the API is accessible from anywhere (GLOBAL) or can only be access from a VPC (PRIVATE).
+   *
+   * @default - GLOBAL
+   */
+  readonly visibility?: string;
```

Comment:
This should be an enum.

Reply:
If it defined as a string in the CfnSpec. 
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-appsync-graphqlapi.html#cfn-appsync-graphqlapi-visibility